src/Personnummer.php

```diff
@@ -123,6 +123,41 @@ public static function getAge($ssn, $includeCoordinationNumber = true)
         return $d1->diff($d2)->y;
     }
 
+    /**
+     * Check if a Swedish social security number is for a male.
+     *
+     * @param string|int $ssn
+     * @param bool       $includeCoordinationNumber
+     *
+     * @return bool
+     * @throws PersonnummerException
+     */
+    public static function isMale($ssn, $includeCoordinationNumber = true)
+    {
+        if (!self::valid($ssn, $includeCoordinationNumber)) {
+            throw new PersonnummerException();
+        }
+
+        $parts       = self::getParts($ssn);
+        $genderDigit = substr($parts['num'], -1);
+
+        return boolval($genderDigit % 2);
```

Comment:
I'm thinking that a `===` or `!!` might be "cheaper" than boolval, but it is basically the same, so not a big deal (not worth changing as of now, hehe).